api/rest/stat_series.md

```diff
@@ -34,69 +35,200 @@ You can find a list of StatisticalVariables with human-readable names [here](/st
 - `unit`: The dcid of the preferred `unit` value.
 - `scaling_factor`: The preferred `scalingFactor` value.
 
-## GET Request
+## How to construct a request to the place statistics time series endpoint
 
-**Examples**
+### Step 1: Assembling the information you will need
 
-```bash
-curl 'https://api.datacommons.org/stat/series?place=geoId/06&stat_var=Count_Person_Male'
-```
+This endpoint requires two arguments and offers four additional optional arguments, as listed:
 
-<iframe width="100%" height="300" src="//jsfiddle.net/datacommonsorg/w32gmo68/8/embedded/" allowfullscreen="allowfullscreen" allowpaymentrequest frameborder="0"></iframe>
+ - `place`: For this parameter, you will need to specify the DCID (the unique ID assigned by Data Commons to each node in the graph) of the place you are interested in.
+ - `stat_var`: The statistical variable whose value you are interested in.
+
+In addition to these required properties, this endpoint also allows for other, optional arguments.
+
```

Comment:
We may want to add a note somewhere about optional args.

Data Commons will return a default values for optional args, if you leave that argument blank. If you "overconstrain" by specifying a unit, scalingFactor, etc that does not exist for the place and variable, there will be no resulting time series.

Feel free to summarize/modify ^

Reply:
I used the language we've had in other parts of the docs here. I also added language to indicate that too many optional args doesn't work out well.

Reply:
This is now split across three different places - can we centralize this, at the top perhaps?

Reply:
I moved it to the top of the section. Let me know what you think!